### PR TITLE
Fix bam rseqc

### PR DESCRIPTION
**Pull request by mahesh-panchal**
## PR checklist

When modules are not in modules list, Nextflow breaks as it cannot find the named channel.

Closes #XXX 

- [x] This comment contains a description of changes (with reason).
- [x] If you've fixed a bug or added code that should be tested, add tests!
- [ ] If you've added a new tool - have you followed the module conventions in the [contribution docs](https://github.com/nf-core/modules/tree/master/.github/CONTRIBUTING.md)
- [ ] If necessary, include test data in your PR.
- [ ] Remove all TODO statements.
- [ ] Emit the `versions.yml` file.
- [ ] Follow the naming conventions.
- [ ] Follow the parameters requirements.
- [ ] Follow the input/output options guidelines.
- [ ] Add a resource `label`
- [ ] Use BioConda and BioContainers if possible to fulfil software requirements.
- Ensure that the test works with either Docker / Singularity. Conda CI tests can be quite flaky:
  - [ ] `PROFILE=docker pytest --tag <MODULE> --symlink --keep-workflow-wd --git-aware`
  - [ ] `PROFILE=singularity pytest --tag <MODULE> --symlink --keep-workflow-wd --git-aware`
  - [ ] `PROFILE=conda pytest --tag <MODULE> --symlink --keep-workflow-wd --git-aware`

### CODE REVIEW

**Comment by mahesh-panchal** [comment]
Bah, was trying to add a test. I forgot to mark as draft.

**Comment by mahesh-panchal** [comment]
What's the nf-test syntax for checking a channel is empty?

File: subworkflows/nf-core/bam_rseqc/main.nf [hunk 1]
```diff
@@ -38,6 +38,7 @@ workflow BAM_RSEQC {
     //
     // Run RSeQC inner_distance.py
     //
+    ch_inner_distance         = Channel.empty()
```
Comment (mashehu) [inline subworkflows/nf-core/bam_rseqc/main.nf:41]:
```suggestion
    ch_innerdistance          = Channel.empty()
```
to make it conform with the other names

File: subworkflows/nf-core/bam_rseqc/main.nf [hunk 2]
```diff
@@ -90,6 +92,7 @@ workflow BAM_RSEQC {
     //
     // Run RSeQC junction_saturation.py
     //
+    ch_junction_saturation        = Channel.empty()
```
Comment (mashehu) [inline subworkflows/nf-core/bam_rseqc/main.nf:95]:
```suggestion
    ch_junctionsaturation         = Channel.empty()
```

File: subworkflows/nf-core/bam_rseqc/main.nf [hunk 3]
```diff
@@ -113,6 +116,7 @@ workflow BAM_RSEQC {
     //
     // Run RSeQC read_duplication.py
     //
+    ch_read_duplication        = Channel.empty()
```
Comment (mashehu) [inline subworkflows/nf-core/bam_rseqc/main.nf:119]:
```suggestion
    ch_readduplication         = Channel.empty()
```

File: subworkflows/nf-core/bam_rseqc/main.nf [hunk 4]
```diff
@@ -138,7 +142,7 @@ workflow BAM_RSEQC {
     }
 
     emit:
-    ch_bamstat                     // channel: [ val(meta), txt ]
+    ch_bamstat                         // channel: [ val(meta), txt ]
 
     ch_inner_distance
```
Comment (mashehu) [inline subworkflows/nf-core/bam_rseqc/main.nf:147]:
```suggestion
    ch_innerdistance
```

File: subworkflows/nf-core/bam_rseqc/main.nf [hunk 5]
```diff
@@ -147,7 +151,7 @@ workflow BAM_RSEQC {
     ch_innerdistance_pdf               // channel: [ val(meta), pdf ]
     ch_innerdistance_rscript           // channel: [ val(meta), r   ]
 
-    ch_inferexperiment             // channel: [ val(meta), txt ]
+    ch_inferexperiment                 // channel: [ val(meta), txt ]
 
     ch_junction_annotation
```
Comment (mashehu) [inline subworkflows/nf-core/bam_rseqc/main.nf:156]:
```suggestion
    ch_junctionannotation
```

File: subworkflows/nf-core/bam_rseqc/main.nf [hunk 6]
```diff
@@ -162,15 +166,15 @@ workflow BAM_RSEQC {
     ch_junctionsaturation_pdf          // channel: [ val(meta), pdf ]
     ch_junctionsaturation_rscript      // channel: [ val(meta), r   ]
 
-    ch_readdistribution            // channel: [ val(meta), txt ]
+    ch_readdistribution                // channel: [ val(meta), txt ]
 
     ch_read_duplication
```
Comment (mashehu) [inline subworkflows/nf-core/bam_rseqc/main.nf:171]:
```suggestion
    ch_readduplication
```

File: subworkflows/nf-core/bam_rseqc/meta.yml [hunk 7]
```diff
@@ -46,91 +46,107 @@ input:
         List of rseqc modules to run
         e.g. [ 'bam_stat', 'infer_experiment' ]
 output:
-  - bamstat_txt:
+  - ch_bamstat:
       type: file
       description: bam statistics report
       pattern: "*.bam_stat.txt"
-  - innerdistance_distance:
+  - ch_inner_distance:
```
Comment (mashehu) [inline subworkflows/nf-core/bam_rseqc/meta.yml:53]:
```suggestion
  - ch_innerdistance:
```

File: subworkflows/nf-core/bam_rseqc/meta.yml [hunk 8]
```diff
@@ -46,91 +46,107 @@ input:
...
       type: file
       description: script to reproduce the plot
       pattern: "*.inner_distance_plot.R"
-  - inferexperiment_txt:
+  - ch_inferexperiment:
       type: file
       description: infer_experiment results report
       pattern: "*.infer_experiment.txt"
-  - junctionannotation_bed:
+  - ch_junction_annotation:
```
Comment (mashehu) [inline subworkflows/nf-core/bam_rseqc/meta.yml:81]:
```suggestion
  - ch_junctionannotation:
```

File: subworkflows/nf-core/bam_rseqc/meta.yml [hunk 9]
```diff
@@ -46,91 +46,107 @@ input:
...
       type: file
       description: Rscript to reproduce the plots
       pattern: "*.r"
-  - junctionannotation_log:
+  - ch_junctionannotation_log:
       type: file
       description: Log file generated by tool
       pattern: "*.log"
-  - junctionsaturation_pdf:
+  - ch_junction_saturation:
```
Comment (mashehu) [inline subworkflows/nf-core/bam_rseqc/meta.yml:113]:
```suggestion
  - ch_junctionsaturation:
```

File: subworkflows/nf-core/bam_rseqc/meta.yml [hunk 10]
```diff
@@ -46,91 +46,107 @@ input:
...
       type: file
       description: Junction saturation R-script
       pattern: "*.r"
-  - readdistribution_txt:
+  - ch_readdistribution:
       type: file
       description: the read distribution report
       pattern: "*.read_distribution.txt"
-  - readduplication_seq_xls:
+  - ch_read_duplication:
```
Comment (mashehu) [inline subworkflows/nf-core/bam_rseqc/meta.yml:129]:
```suggestion
  - ch_readduplication:
```

**Pull request merged**